### PR TITLE
add ventoy guide for bootableusb.md

### PR DESCRIPTION
**Pull request by CoolPenguin27**
This pr updates the [bootableusb.md](https://github.com/chrultrabook/docs/blob/main/docs/bootableusb.md) page to have a Ventoy guide, and also does the following:

Replaces "Flashing on other opperating systems" section 
Changes USB references to external drive
Changes Step 3 note in "Prerequisites" section into a warning

### CODE REVIEW

File: docs/bootableusb.md [hunk 1]
```diff
@@ -6,17 +6,18 @@ nav_order: 25
...
+{: .warning }
+Flashing Ventoy or ISO's to an external drive will **wipe all data** on said drive. Back up anything if you have to.
+
 ### Prerequisites
 1. Another PC or laptop with at least 8GB storage
 2. A internet connection
-3. A USB drive or SD card thats at least 8 gigabytes.
-      - You will be erasing this drive, backup anything if you need to do so.
-
+3. A external drive like a USB drive or SD card thats at least 8 gigabytes.
```
Comment (WeirdTreeThing) [inline docs/bootableusb.md:15]: that's

File: docs/bootableusb.md [hunk 2]
```diff
@@ -6,17 +6,18 @@ nav_order: 25
...
 
 ### Downloading a ISO
 1. Determine what OS you want.
    - For Linux, keep in mind Ubuntu and any derivatives will not work.
```
Comment (WeirdTreeThing) [inline docs/bootableusb.md:19]: not entirely true, we need more consistency about that throughout the entire docs

**Review by WeirdTreeThing — CHANGES_REQUESTED** [review]

File: docs/bootableusb.md (hunk 2 above)
Comment (CoolPenguin27) [reply docs/bootableusb.md:19]: What about changing it to 4gb for Linux, 8gb for Windows?

Comment (WeirdTreeThing) [reply docs/bootableusb.md:19]: smh github, that comment was supposed to be on the line for ubuntu derivatives not being supported

Comment (CoolPenguin27) [reply docs/bootableusb.md:19]: for clarification, what is the state of ubuntu for future commits?

Comment (WeirdTreeThing) [reply docs/bootableusb.md:19]: after running the audio script, they should work.

**Review by meghan06 — APPROVED** [review]
looks good to me

File: docs/bootableusb.md (hunk 2 above)
Comment (CoolPenguin27) [reply docs/bootableusb.md:19]: can this be done on a separate pr?

Comment (WeirdTreeThing) [reply docs/bootableusb.md:19]: sure

**Review by WeirdTreeThing — APPROVED** [review]

**Pull request merged**